samples/outlook/99-preview-apis/get-conversation-index.yaml

```diff
@@ -0,0 +1,63 @@
+order: 6
+id: outlook-get-conversation-index
+name: Get the position of a message in a conversation (Message Compose)
+description: Retrieves the Base64-encoded position of the current message in a conversation thread.
+host: OUTLOOK
+api_set:
+    Mailbox: preview
+script:
+    content: |
+        $("#get-conversation-index").click(getConversationIndex);
+
+        function getConversationIndex() {
+          // This snippet returns the Base64-encoded position of the current message in a conversation thread (PR_CONVERSATION_INDEX).
+          // The API call is supported on a message being composed and isn't supported on read items.
+          const conversationIndex = Office.context.mailbox.item.conversationIndex;
+          if (conversationIndex) {
+            console.log("Position in the conversation thread: " + conversationIndex);
+          }
+          else
+          {
+            console.log("The current message doesn't belong to a conversation thread.");
+          }
+        }
```

Comment:
```suggestion
        $("#get-conversation-index").click(getConversationIndex);

        function getConversationIndex() {
          // This snippet returns the Base64-encoded position of the current message in a conversation thread (PR_CONVERSATION_INDEX).
          // The API call is supported on a message being composed and isn't supported on read items.
          Office.context.mailbox.item.getConversationIndexAsync((result) => {
             if (result.status === Office.AsyncResultStatus.Failed) {
                console.log(result.error.message);
                return;
             }
             
             const conversationIndex = result.value;
             if (conversationIndex) {
                console.log("Position in the conversation thread: " + conversationIndex);
             }
             else
             {
                console.log("The current message doesn't belong to a conversation thread.");
             }
          });
        }
```

Reply:
Thanks for the update here, Sam! I accidentally made the changes manually instead of commit through this PR. They should be the same though.